test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,177 @@ Feature: Bind an application to a service
+            """
+        Then Secret "sbr-csv-attribute" must contain data from resource type "Backserv" cr name "demo-backserv-cr-1" and jsonpath "{.spec.svcName}"
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "False"
+
+    Scenario: Create intermidiate secret using information mentioned in x-descriptors of backend service CSV
+        Given Backend service "some-backend-service.v0.2.0" CSV is installed
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ClusterServiceVersion
+            metadata:
+                name: some-backend-service.v0.2.0
+            spec:
+                displayName: Some Backend Service
+                install:
+                    strategy: deployment
+                customresourcedefinitions:
+                    owned:
+                        - name: backservs.service.example.com
+                          version: v1
+                          kind: Backserv
+                          statusDescriptors:
+                            - description: Name of the Secret to hold the DB user and password
+                              displayName: DB Password Credentials
+                              path: secret
+                              x-descriptors:
+                              - urn:alm:descriptor:io.kubernetes:Secret
+                              - binding:env:object:secret:username
+                              - binding:env:object:secret:password
+                            - description: Name of the ConfigMap to hold the DB config
+                              displayName: DB Config Map
+                              path: configmap
+                              x-descriptors:
+                              - urn:alm:descriptor:io.kubernetes:ConfigMap
+                              - binding:env:object:configmap:db_host
+                              - binding:env:object:configmap:db_port
+            """
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The CR "demo-backserv-cr-2" for kind "Backserv" is present
```

Comment:
Do we really need to specify kind in step? How about saying something like:

```
Resource is applied
```

With such step we can apply anything - configmaps, secrets as well.

Reply:
If we don't specify kind here then it will be hard to assert if the resource is created or not. https://github.com/redhat-developer/service-binding-operator/pull/620/files#diff-ec891afbdb0cf5fd075cbf5ca535a45fR329

Reply:
We might need to look into better assertion method here then.

Reply:
You can get kind info from the submitted yaml content.

Reply:
Done.